src/client/main.cpp

```diff
@@ -2,6 +2,87 @@
 #include <memory>
 #include <string>
 
+#include "helper/parser.h"
+
+using namespace std;
+
+string loggedInUsername = "";
+
+void handleCreateHost(vector<string> input) {
```

Comment:
Prefer pass-by-reference (i.e. vector<string>& input), that way you avoid copying the vector

Reply:
changed.